packages/core/src/modules/connections/services/ConnectionService.ts

```diff
@@ -550,22 +560,22 @@ export class ConnectionService {
    * @param connectionId the connection record id
    * @returns The connection record or null if not found
    */
-  public findById(connectionId: string): Promise<ConnectionRecord | null> {
-    return this.connectionRepository.findById(connectionId)
+  public findById(agentContext: AgentContext, connectionId: string): Promise<ConnectionRecord | null> {
+    return this.connectionRepository.findById(agentContext, connectionId)
   }
 
   /**
    * Delete a connection record by id
    *
    * @param connectionId the connection record id
    */
-  public async deleteById(connectionId: string) {
-    const connectionRecord = await this.getById(connectionId)
-    return this.connectionRepository.delete(connectionRecord)
+  public async deleteById(agentContext: AgentContext, connectionId: string) {
+    const connectionRecord = await this.getById(agentContext, connectionId)
+    return this.connectionRepository.delete(agentContext, connectionRecord)
   }
 
-  public async findSingleByQuery(query: { did: string; theirDid: string }) {
-    return this.connectionRepository.findSingleByQuery(query)
+  public async findByDids(agentContext: AgentContext, query: { ourDid: string; theirDid: string }) {
```

Comment:
Is there a reason why we call it `ourDid`? I believe ACA-Py uses `my_did`? I have no preference, more just curious.

Reply:
Not sure, @jakubkoci added this